### PR TITLE
[FIX] Fixed test_start and uncollected reboot

### PR DESCRIPTION
**Pull request by jteehan**
Purpose or Intent
=================

__Fixing__ 

Can't use wait for vm state change as azure may transition through several and it the timing is off, this will fail.  Usually fails.

Works best with new Azure Mgmt PR 63

{{pytest: cfme/tests/cloud/test_instance_power_control.py -k "test_start" --long-running --use-provider azure}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 04796bc131aa838e5aa314ca7398a36bf119aba1:

`cfme/tests/cloud/test_instance_power_control.py`:
- :bangbang: Line 377:64: **E231** *missing whitespace after ':'*
- :bangbang: Line 377:64: **E701** *multiple statements on one line (colon)*
- :bangbang: Line 377:66: **E901** *SyntaxError: invalid syntax*

Please, rectify these issues :smirk: .

Commit inspection report:
**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a0122e69457b42b3382a19fe6d193eb4e60ec0ab:

Everything seems all right from lint perspective :smile: .

Commit inspection report:
**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
PRT failures are both relevant I think

File: cfme/tests/cloud/test_instance_power_control.py [hunk 1]
```diff
@@ -192,8 +192,11 @@ def test_start(
     Metadata:
         test_flag: power_control, provision
     """
-    testing_instance.wait_for_vm_state_change(
-        desired_state=testing_instance.STATE_OFF, timeout=720, from_details=True)
+    if provider.type == 'azure':
```
Comment (psav) [inline cfme/tests/cloud/test_instance_power_control.py]: Isn't this covered by the verify_vm_stopped fixture?

File: cfme/tests/cloud/test_instance_power_control.py [hunk 2]
```diff
@@ -265,8 +269,9 @@ def test_suspend(
     Metadata:
         test_flag: power_control, provision
     """
-    testing_instance.wait_for_vm_state_change(
-        desired_state=testing_instance.STATE_ON, timeout=720, from_details=True)
+    if provider.type != 'azure':
```
Comment (psav) [inline cfme/tests/cloud/test_instance_power_control.py]: This will never get run against azure due to the uncollect on line 264, there is also a verify_vm_running fixture here, does that not do the job? If not, we should consider fixing that up instead.

File: cfme/tests/cloud/test_instance_power_control.py [hunk 3]
```diff
@@ -332,14 +337,15 @@ def test_terminate(setup_provider_funcscope, provider, testing_instance, soft_as
...
```
Comment (psav) [inline cfme/tests/cloud/test_instance_power_control.py]: Same comments as above here and then below

File: utils/log.py [hunk 4]
```diff
@@ -557,9 +550,19 @@ def process(self, msg, kwargs):
 
 perflog = Perflog()
 
-# Capture warnings to the cfme logger using the warnings.showwarning hook
-warnings.showwarning = _showwarning
-warnings.simplefilter('default')
+
+def _configure_warnings():
```
Comment (psav) [inline utils/log.py]: Something perhaps is wrong with your commit here, these shouldn't be part of this PR

File: cfme/tests/cloud/test_instance_power_control.py (hunk 1 above)
Comment (lkhomenk) [inline cfme/tests/cloud/test_instance_power_control.py]: verify_vm_stopped gets state from mgmt from background like running,stopped,deleted, which doesn't reflect  vm state in web ui. and wait_for_vm_state_change simulates web ui, as we need to update page and/or refresh relationships

**Comment by jteehan** [comment]
Mine is not a dupe of https://github.com/ManageIQ/integration_tests/pull/3326 as he got his in first.  Closing and will work with Leonid on the review.
